ownCloudAppShared/Client/Sharing/ShareViewController.swift

```diff
@@ -123,6 +126,19 @@ open class ShareViewController: CollectionViewController, SearchViewControllerDe
 		self.item = (item != nil) ? item! : ((location != nil) ? try? clientContext.core?.cachedItem(at: location!) : nil)
 		self.mode = mode
 		self.completionHandler = completion
+		
+		// Item section
+		let itemSectionContext = ClientContext(with: clientContext, modifier: { context in
```

Comment:
This could be moved to within the `if let item {` block below as `itemSectionContext` is only needed there.